actors/builtin/multisig/multisig_actor.go

```diff
@@ -39,7 +39,7 @@ func (a *MultiSigActor) State(rt vmr.Runtime) (vmr.ActorStateHandle, MultiSigAct
 }
 
 type ConstructorParams struct {
-	Signers     []addr.Address
```

Comment:
Thanks for reminding me that CI needs a gofmt check. https://github.com/filecoin-project/specs-actors/issues/42